### PR TITLE
update chainID to prevent replay attacks

### PR DESCRIPTION
**Pull request by lastwallfacer**

### CODE REVIEW

**Comment by ligi** [comment]
don't use 10001 - it is already used: https://github.com/ethereum-lists/chains/blob/master/_data/chains/eip155-10001.json
so you will expose your chain to replay attacks still
also you will never be listed at e.g. chainlist.org
choose another one ..

**Comment by mrtestyboy781** [comment]
'PoW' in ascii is 50 6F 57 -> 5271383 in decimal

Make the chainId=5271383

**Comment by rshi0614** [comment]
anyone from the core ethw team doing this?

File: params/config.go [hunk 1]
```diff
@@ -57,7 +57,7 @@ var CheckpointOracles = map[common.Hash]*CheckpointOracleConfig{
 var (
 	// MainnetChainConfig is the chain parameters to run a node on the main network.
 	MainnetChainConfig = &ChainConfig{
-		ChainID:             big.NewInt(1), //10001
+		ChainID:             big.NewInt(10001),
```
Comment (zquestz) [inline params/config.go]: This chainID is currently used by smartBCH testnet.

I would suggest a different value. Perhaps 12001.

Comment (lastwallfacer) [reply params/config.go]: 八六九

to make a fortune, the fork will go smoothly and last forever

ate 69 for the lulz

**Review by Koubek11 — APPROVED** [review]

**Pull request closed without merging**